saturn-rapids/environment.yml

```diff
@@ -0,0 +1,31 @@
+name: saturn
+channels:
+- nvidia
+- rapidsai
+- numba
+- defaults
+- conda-forge
+dependencies:
+- blas=*=mkl
+- bokeh
+- cvxpy
+- dask-ml
+- dask=2.30.0
+- distributed=2.30.1
+- ipykernel
+- ipywidgets
+- matplotlib
+- numpy
```

Comment:
I think `numba` should be here too, because rapids allows UDF building with numba

Reply:
I was going back and forth on whether to explicitly include dependencies that are included already. For instance by including rapids we get:

```python
dependencies: 
  - cudatoolkit 10.1.*
  - cudf 0.17.*
  - cugraph 0.17.*
  - cuml 0.17.*
  - cupy >7.1.0,<9.0.0a0
  - cusignal 0.17.*
  - cuspatial 0.17.*
  - custreamz 0.17.*
  - cuxfilter 0.17.*
  - dask-cuda 0.17.*
  - nccl >=2.7.8.1,<3.0a0
  - networkx >=2.3
  - numba >=0.51.2
  - numpy >=1.17.3
  - nvtx >=0.2.1,<0.3
  - pickle5
  - python >=3.7,<3.8.0a0
  - python_abi 3.7.* *_cp37m
  - rapids-xgboost 0.17.*
  - rmm 0.17.*
  - setuptools >=49,<50
```

Reply:
Oh I see, I like it as-is then. I didn't realize rapids pulled in numba